kedro/templates/project/{{ cookiecutter.repo_name }}/.coveragerc

```diff
@@ -1,3 +1,5 @@
+[run]
+branch=True
```

Comment:
I wouldn't want to turn on branch coverage by default for user projects. I think this has has to be a conscious user choice

Reply:
Don't you think it's a good practise to encourage users to enable code coverage before they start building things? Users can then choose to disable the flag if they don't wish to use it.

In practise it is seen, that if this is switched off, user's won't notice and only very few might enable them.

 I'm thinking a bigger number will keep this enabled, as opposed to switching it on as a conscious choice.